frontend/src/components/static_pages/about.css

```diff
@@ -1,154 +1,146 @@
+.mainContainer {
+  padding: 0px 50px 25px 50px;
+}
 #aboutPageTextRow {
-  padding-left: 133px;
-  padding-right: 133px;
+  margin: 0px;
   display: block;
 }
-
 .sub-heading {
   font-size: 24px;
+  padding: 0px;
+  width: 100%;
 }
 
 .about-people-row {
-  margin-bottom: 20px;
+  margin: 40px 0px;
 }
+
 .images {
-  margin-bottom: 60px;
-  margin-left: 120px;
-  margin-right: 40px;
+  display: flex;
+  align-items: center;
 }
 
 .images img {
-  width: 330px;
+  width: 85%;
   border: 5px solid #d3d3d3;
 }
 
-#kim-image {
-  margin-top: 90px;
-  margin-bottom: 30px;
-  height: 350px;
-}
-
-#pierre-image {
-  margin-top: 30px;
-  margin-bottom: 30px;
-  height: 350px;
-}
-
-#shirley-image {
-  margin-top: 30px;
-  height: 355px;
-}
-
 .about-heading {
-  margin-left: 120px;
-  margin-bottom: 10px;
-  padding: 15px 0px;
+  margin: auto;
+  font-size: 48px;
   font-weight: bold;
+  width: 100%;
+  text-align: center;
 }
 
 .people-heading {
-  margin-top: 45px;
-  font-size: 48px;
+  font-size: 40px;
   font-weight: bold;
   display: inline-block;
+  width: 100%;
 }
 
-/* tablet */
+@media (min-width: 2000px) {
+  .mainContainer {
+    max-width: 2000px;
+  }
 
-@media all and (min-width: 768px) and (max-width: 1023px) {
-  #aboutPageTextRow {
-    padding-left: 20px;
-    padding-right: 20px;
+  p {
```

Comment:
we have the same problem here with the `p` as we did with the container -> this will take affect for all `p` tags across the app. 

One way to solve for this is to nest the p one within the .main-container like so

```css
.mainContainer {
    max-width: 2000px;

    p {
      font-size: 32px;
      margin-bottom: 10px;
    }

  }
````

this way it will only take affect for when the `main-container` class is present

Reply:
I did the changes for the p tag but I didn't used the CSS nesting that you suggested, I used 
.mainContainer p {
      font-size: 32px;
      margin-bottom: 10px;
}

the nesting that you suggested was not changing the font size.... I hope this is good! :crossed_fingers: